algorithms/indexing/stills_indexer.py

```diff
@@ -790,10 +789,6 @@ def refine(self, experiments, reflections):
 """Mixin class definitions that override the dials indexing class methods specific to stills"""
 
 
-class StillsIndexerKnownOrientation(IndexerKnownOrientation, StillsIndexer):
```

Comment:
Did your script to find dead code include other repos?  Please keep this class.
https://github.com/cctbx/cctbx_project/blob/master/xfel/command_line/xtc_process.py#L1349-L1353

Reply:
:+1: